base/test/middleware/jwt_test.mustache

```diff
@@ -0,0 +1,37 @@
+const fetch = require('node-fetch');
+const jwt   = require('jsonwebtoken');
```

Comment:
Do we need to align assignments?  This is probably one of "those" conversations, but we haven't done that on any other projects so to be safe I think let's not introduce the style here, but let's discuss it.